source-monday/source_monday/api.py

```diff
@@ -32,10 +31,12 @@
 # However, the underlying GraphQL query will fetch items in smaller batches to balance
 # complexity and other API limitations.
 BOARDS_PER_ITEMS_PAGE = 25
+BOARDS_PER_BOARD_PAGE = 100
```

Comment:
TODO: change this back. Did not mean to commit this line. Was testing a few other exceptions around complexity limits.